src/util/otpReminder.tsx

```diff
@@ -0,0 +1,92 @@
+import type { EdgeAccount } from 'edge-core-js'
+import * as React from 'react'
+import { Keyboard } from 'react-native'
+import { sprintf } from 'sprintf-js'
+
+import { ButtonsModal } from '../components/modals/ButtonsModal'
+import { Airship } from '../components/services/AirshipInstance'
+import { lstrings } from '../locales/strings'
+
+const OTP_REMINDER_MILLISECONDS = 7 * 24 * 60 * 60 * 1000
+const OTP_REMINDER_STORE_NAME = 'app.edge.login'
+const OTP_REMINDER_KEY_NAME_LAST_OTP_CHECKED = 'lastOtpCheck'
+const OTP_REMINDER_KEY_NAME_DONT_ASK = 'OtpDontAsk'
+
+/**
+ * Check and show the 2fa reminder on login.
+ */
+export async function showOtpReminder(account: EdgeAccount): Promise<void> {
+  const { otpKey, dataStore, username, created } = account
+  if (username == null) return
+  if (otpKey != null) return
+
+  const [dontAsk, lastOtpCheckString]: [string | null, string | null] = (await Promise.all([
+    dataStore.getItem(OTP_REMINDER_STORE_NAME, OTP_REMINDER_KEY_NAME_DONT_ASK).catch(() => null),
+    dataStore.getItem(OTP_REMINDER_STORE_NAME, OTP_REMINDER_KEY_NAME_LAST_OTP_CHECKED).catch(() => null)
+  ])) as any
```

Comment:
This `as any` was to work around a bizarre TypeScript bug in the edge-login-ui-rn repo. It may not be needed here.

Reply:
Seems no longer needed